perf/variant.py

```diff
@@ -26,18 +26,14 @@ def __init__(self, name: str, semgrep_core_extra: str, semgrep_extra: str = ""):
     SemgrepVariant("max-cache", "-opt_max_cache"),
     SemgrepVariant("no-bloom", "-no_bloom_filter"),
     SemgrepVariant("no-gc-tuning", "-no_gc_tuning"),
+    SemgrepVariant("no-filter-irrelevant-rules", "-no_filter_irrelevant_rules"),
```

Comment:
Does this change belong to this PR?

Reply:
Not really, but I had to make this change to prevent some tests from failing. See @emjin 's comment here https://github.com/returntocorp/semgrep/pull/3844

Reply:
I'm going to just merge this, in the future will make separate PR in cases like this

Reply:
Totally fine, it just looked strange.